src/Integration/State/StateManager.cs

```diff
@@ -127,8 +129,15 @@ private void OnIsBusyChanged(bool isBusy)
             this.IsBusyChanged?.Invoke(this, isBusy);
         }
 
-        private void OnBindingStateChanged()
+        private void OnBindingStateChanged(bool isBound)
```

Comment:
You can keep the field if you will use it as a condition to raise the event or not i.e.

private void OnBindingStateChanged(bool isBound)
{
    if (this.previouslyBound != isBound)
    {
        this.previouslyBound = isBound;
        this.BindingStateChanged?.Invoke(this, isBound);
     }
}
}